Configurations/windows-makefile.tmpl

```diff
@@ -464,23 +464,22 @@ depend:
 
 install_sw: install_dev install_engines install_modules install_runtime
 
-uninstall_sw: uninstall_runtime uninstall_modules uninstall_engines uninstall_dev
+uninstall_sw: uninstall_runtime uninstall_fips uninstall_modules uninstall_engines uninstall_dev
 
 install_docs: install_html_docs
 
 uninstall_docs: uninstall_html_docs
 
 install_fips: install_sw
 	@$(ECHO) "*** Installing FIPS module configuration"
-	@$(ECHO) "fipsinstall $(DESTDIR)$(MODULESDIR)/$(FIPSMODULENAME).cnf"
-	@openssl fipsinstall -module $(DESTDIR)$(MODULESDIR)/$(FIPSMODULENAME) \
-		-out $(DESTDIR)$(MODULESDIR)/$(FIPSMODULENAME).cnf \
-		-macopt "key:$(FIPSKEY)"
+	"$(SRCDIR)\util\wrap.pl" "$(SRCDIR)\apps\openssl" fipsinstall \
```

Comment:
`"$(SRCDIR)\util\wrap.pl" "$(SRCDIR)\apps\openssl"` should be `"$(BLDDIR)\apps/openssl"`

(uitl/wrap.pl is Unix only)

Reply:
I thought you had intended the Perl Script to be a platform independent wrapper, when you created it (contrary to shlib_wrap.sh). Even on windows, it's not a no-op, it sets up the module paths. So thought I might as well leave it for symmetry and future extensibility. But if you think it's pointless here, I can remove it.

https://github.com/openssl/openssl/blob/0cc0164d193f6071a9d06b2116a410f8139a7e3c/util/wrap.pl#L9-L12

Although I have to admit I _do_ like the way it treats the Windows case:

https://github.com/openssl/openssl/blob/0cc0164d193f6071a9d06b2116a410f8139a7e3c/util/wrap.pl#L22-L27

Reply:
Ah, duh, you're right.  I had just forgotten...